app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java

```diff
@@ -132,19 +134,19 @@ public boolean onQueryTextChange(String s) {
       }
     });
 
-    MenuItemCompat.setOnActionExpandListener(searchMenuItem,
-        new MenuItemCompat.OnActionExpandListener() {
-          @Override
-          public boolean onMenuItemActionExpand(MenuItem item) {
-            return false;
-          }
+    //replaced deprecated method
+    searchMenuItem.setOnActionExpandListener(new MenuItem.OnActionExpandListener() {
```

Comment:
Convert to lambda if possible.

Reply:
Sorry. The above can't be converted to a lambda as It overrides more than one method.

Reply:
Okay :smile: